solutions/devsprint-bruno-ramos-2/DeliveryAppChallenge/Modules/Product/Home/HomeViewController.swift

```diff
@@ -1,33 +1,59 @@
 //
-//  ViewController.swift
+//  HomeViewController.swift
 //  DeliveryAppChallenge
 //
-//  Created by Rodrigo Borges on 25/10/21.
+//  Created by Leonardo Mendes on 30/01/22.
 //
 
+import Foundation
 import UIKit
 
-class HomeViewController: UIViewController {
+/// Protocol used to intermediates communication from Presenter to ViewController ( Presenter ----`viewModel` ----> ViewController )
+protocol HomeDisplayLogic: AnyObject {
+    func displayViewModel(viewModel: HomeUseCase.FetchData.ViewModel)
```

Comment:
Aqui na viewcontroller podemos também explicar qual vai ser o display nesse ciclo, tornando as nossas instruções mais legíveis
```suggestion
    func displayRestaurantListl(viewModel: HomeUseCase.FetchData.ViewModel)
```

Reply:
Corrigido